parachain/pallets/inbound-queue/fixtures/src/send_token.rs

```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2023 Snowfork <hello@snowfork.com>
+// Generated, do not edit!
+// See ethereum client README.md for instructions to generate
+
+use crate::InboundQueueFixture;
+use hex_literal::hex;
+use snowbridge_beacon_primitives::CompactExecutionHeader;
+use snowbridge_core::inbound::{Log, Message, Proof};
+use sp_std::vec;
+
+pub type SendToken = InboundQueueFixture;
```

Comment:
I think its a bit unnecessary to add this alias for each fixture.

Reply:
We should rather add a doc comment (only if necessary - the function name may be enough) saying that the returned fixture contains a send token message.